neuro_san/api/grpc/agent_pb2.py

```diff
@@ -11,13 +11,23 @@
 # END COPYRIGHT
 # -*- coding: utf-8 -*-
 # Generated by the protocol buffer compiler.  DO NOT EDIT!
+# NO CHECKED-IN PROTOBUF GENCODE
 # source: neuro_san/api/grpc/agent.proto
-# Protobuf Python Version: 4.25.1
+# Protobuf Python Version: 6.31.1
```

Comment:
Redid ./neuro_san/api/scripts/do_generate.sh on Python 3.12 with protobuf==6.31.1.
Here are some testing results:
* agent_service.json is generated exactly the same way by the golang tools, as far as I can tell.
* new protobuf 6 server with new protobuf 6 client is fine
* new protobuf 6 server with old protobuf 4 client is fine
* old protobuf 4 server with new protobuf 6 client is fine